lang/en.js

```diff
@@ -0,0 +1,123 @@
+            'blocks': "You have unsaved code. Using blocks will change your code. You may lose your changes. Do you want to continue?"
+          },
+          'code': {
+            'start': "# Add your Python code here. E.g.\n" +
+                     "from microbit import *\n\n\n" +
+                     "while True:\n" +
+                     "    display.scroll('Hello, World!')\n" +
+                     "    display.show(Image.HEART)\n" +
+                     "    sleep(2000)\n"
+          },
+          'drop': {
+            'python': 'Extracted from a Python file',
+            'hex': 'Extracted from a hex file'
+          },
+          'messagebar': {
+            'title': 'This editor is out of date!',
+            'link': 'Click here',
+            'instructions': 'to open the latest version.',
+            'close': 'Hide'
+          },
+          'load': {
+            'title': 'Load Script or HEX file',
+            'instructions': 'Drag and drop a .hex or .py file in here to open it.',
+            'submit': 'Load',
+            'toggle-drop': 'Or drag and drop a file',
+            'toggle-file': 'Or pick a file'
+          },
+          'static-strings':{
+            'buttons':{
+              'command-download':{
+                'title': 'Download a hex file to flash onto your micro:bit',
+                'label': 'Download'
+              },
+              'command-files':{
+                'title':'Load/Save files',
+                'label':'Load/Save'
+              },
+              'command-serial':{
+                'title':'Connect your micro:bit via serial',
+                'label':'Open Serial'
+              },
+              'command-connect':{
+                'title':'Connect to your micro:bit',
+                'label':'Connect'
+              },
+              'command-options':{
+                'title':'Change the editor settings',
+                'label':'Options'
+              },
+              'command-blockly':{
+                'title':'Click to create code with blockly',
+                'label':'Blockly'
+              },
+              'command-snippet':{
+                'title':'Click to select a snippet (code shortcut)',
+                'label':'Snippets'
+              },
+              'command-help':{
+                'title':'Discover helpful resources',
+                'label':'Help'
+              },
+              'command-language':{
+                'title':'Select a language',
+                'label':'Language'
+              },
+              'command-share':{
+                'title':'Create a link to share your script',
+                'label':'Share'
+              },
+              'zoom-in':{
+                'title':'Zoom in'
+              },
+              'zoom-out':{
+                'title':'Zoom out'
+              }
+            },
+          'script-name':{
+            'label': 'Script Name'
+            }
+          }
+    };
```

Comment:
The filesystem and webusb branches have modified this values, so make sure you compare the old object from editor.html and make sure all the entries are here (and in the other translations) as well.